infra/base-images/base-builder/install_rust.sh

```diff
@@ -19,3 +19,11 @@ curl https://sh.rustup.rs | sh -s -- -y --default-toolchain=nightly --profile=mi
 cargo install cargo-fuzz && rm -rf /rust/registry
 # Needed to recompile rust std library for MSAN
 rustup component add rust-src --toolchain nightly
+
+(
+unset CFLAGS
+unset CXXFLAGS
+# so that compile_libfuzzer can copy, without knowing the clang version
+rm /usr/local/lib/clang/*/lib/linux/libclang_rt.fuzzer-*
```

Comment:
This uses an older version of libfuzzer for some of our projects right?
This can be a huge hassle for us and I don't think we want it.